runelite-client/src/main/java/net/runelite/client/plugins/nightmarezone/NightmareZoneConfig.java

```diff
@@ -135,11 +147,33 @@ default Color absorptionColorAboveThreshold()
 		keyName = "absorptioncolorbelowthreshold",
 		name = "Color below threshold",
 		description = "Configures the color for the absorption widget when below the threshold",
-		position = 10
+		position = 11
 	)
 	default Color absorptionColorBelowThreshold()
 	{
 		return Color.RED;
 	}
 
+	@ConfigItem(
+		keyName = "pointsperhour",
+		name = "Show points per hour",
+		description = "Toggles weither or not showing points per hour.",
```

Comment:
The existing descriptions do not have trailing periods. Match that?

Reply:
Good catch, pushing a fix now